diff --git a/app.toml b/app.toml
index 60349a4..c10d111 100644
--- a/app.toml
+++ b/app.toml
@@ -8,7 +8,7 @@
 # The minimum gas prices a validator is willing to accept for processing a
 # transaction. A transaction's fees must meet the minimum of any denomination
 # specified in this config (e.g. 0.25token1,0.0001token2).
-minimum-gas-prices = ""
+minimum-gas-prices = "0stake"

 # The maximum gas a query coming over rest/grpc may consume.
 # If this is set to zero, the query can consume an unbounded amount of gas.
